src/Sarif.Converters/PREFastConverter.cs

```diff
@@ -106,15 +107,53 @@ private Result CreateResult(Defect defect)
             };
 
             result.Locations.Add(location);
+
             SetProbability(defect, result);
             SetRank(defect, result);
-
             SetCategories(defect, result);
+
             GenerateCodeFlows(defect, result);
+            GenerateSuppressions(defect.AdditionalInfo, result);
 
             return result;
         }
 
+        private void GenerateSuppressions(AdditionalInfo additionalInfo, Result result)
+        {
+            if (additionalInfo == null) { return; }
+
+            result.Suppressions = new List<Suppression>();
+            result.Suppressions.Add(GenerateSuppression(additionalInfo));
+        }
+
+        internal static Suppression GenerateSuppression(AdditionalInfo additionalInfo)
+        {
+            // If this sentinel key does not exist, we have not suppression data.
+            if (!additionalInfo.ContainsKey("SuppressedMatch")) return null;
```

Comment:
>null [](start = 71, length = 4)

BUG: This leaves the value of `result.Suppressions` as a list containing a single element whose value is `null`. I suggest instead (in `GenerateSuppressions` above):

```
if (additionalInfo == null) { return; }

Suppression suppression = GenerateSuppression(additionalInfo);
if (suppression == null) { return; }

result.Suppressions = new List<Suppression> { suppression };
```